usaspending_api/disaster/v2/views/object_class/loans.py

```diff
@@ -0,0 +1,68 @@
+from django.db.models import Q, Sum, F, Value, Case, When
+from django.db.models.functions import Coalesce
+from rest_framework.response import Response
+
+from usaspending_api.awards.models import FinancialAccountsByAwards
+from usaspending_api.common.cache_decorator import cache_response
+from usaspending_api.disaster.v2.views.disaster_base import (
+    DisasterBase,
+    LoansPaginationMixin,
+    LoansMixin,
+    FabaOutlayMixin,
+)
+from usaspending_api.disaster.v2.views.object_class.spending import universal_annotations, construct_response
+
+
+class ObjectClassLoansViewSet(LoansMixin, LoansPaginationMixin, FabaOutlayMixin, DisasterBase):
+    """View to implement the API"""
```

Comment:
Very minor, but can you make this more meaningful? I know this was (is?) on other endpoints but since it is the description of the endpoint on `api.usaspending.gov` it should be slightly more descriptive of the endpoint.